src/VisualStudio/CSharp/Impl/CSharpVSResources.resx

```diff
@@ -516,4 +516,7 @@
   <data name="When_possible" xml:space="preserve">
     <value>When possible</value>
   </data>
+  <data name="Warn_on_invalid_string_dot_format_calls" xml:space="preserve">
+    <value>Warn on invalid string.Format method calls</value>
```

Comment:
* "Report uses of 'string.Format' with invalid placeholders"
* "Report invalid placeholders in 'string.Format' expressions"

@Pilchie why does this need to be an option at all? When would someone *not* want to fix this?